auth/src/main/res/values/colors.xml

```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="authui_colorPrimary">#3E50B4</color>
-    <color name="authui_colorPrimaryDark">#2F3E9E</color>
-    <color name="authui_colorAccent">#3E50B4</color>
-    <color name="authui_inputTextColor">#FFFFFF</color>
-    <color name="authui_inputTextColorDark">#000000</color>
+    <color name="colorPrimary">#3F51B5</color>
+    <color name="colorPrimaryDark">#303F9F</color>
+    <color name="colorAccent">#FF4081</color>
```

Comment:
@samtstern Yup and yup! I'm happy to change the colors back to what they were since they shouldn't matter; what I really want is a way to error if the dev doesn't override our colors, but I don't think that's possible.